structuredefinitions/Extension-England-FGMRemovalReason.xml

```diff
@@ -38,7 +40,8 @@
   <differential>
     <element id="Extension.url">
       <path value="Extension.url" />
-      <fixedUri value="https://fhir.nhs.uk/England/StructureDefinition/Extension-England-FGMRemovalReason" />
+      <fixedUri
```

Comment:
Not sure what you tried to change here but should be just over one line (new line has been introduced)